apps/schema/src/components/TableEditModal.vue

```diff
@@ -1,156 +1,196 @@
 <template>
-  <!-- when authorisation graphqlError show login-->
-  <Spinner v-if="loading" />
-  <div v-else-if="showLogin">
-    <MessageError v-if="graphqlError">{{ graphqlError }}</MessageError>
-    <SigninForm @login="loginSuccess" @cancel="cancel" />
-  </div>
-  <!-- when update succesfull show result before close -->
+  <IconAction
+    v-if="!show"
```

Comment:
It is a better practice to put the truthy conditions first. It reduces the cognitive load, switching the logic around would do the trick

Reply:
ok